src/assets/Comments.jsx

```diff
@@ -11,15 +11,15 @@ export default function Comments() {
     const [lastId, setLastId] = useState(5)
     const [activeComment, setActiveComment] = useState(null)
     const [activeModal, setActiveModal] = useState(null)
-
-    React.useEffect(() => {
+    
+    useEffect(() => {
         localStorage.setItem('comments', JSON.stringify(commentData))
     }, [commentData])
 
-    function sortComments() {
-        setCommentData(prevData => prevData.sort((a, b) => b.score - a.score))
-    };
-
+    function sortByScore(array) {

```

Comment:
This is not a good implementation because the "sort" function changes the original array - doesn't create a new one. And it's a bad idea to create a function that mutates (changes) its arguments. So that's why I recommended for the "reusable" function to be just the one used for "sort" itself:
`sortByScore = (a, b) => b.score - a.score`
or 
`function sortByScore(a, b) { return b.score - a.score; }`
And then use that function as:
`newCommentData = prevCommentData.map(com => {...}).sort(sortByScore);`
This way you're reusing the "sort mechanism" function but not the sort itself.